test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -122,6 +123,26 @@ call_log_manager::process_call_log_entry(json call_log_entry)
             }
             break;
         }
+        case api_method::begin_transaction: {
```

Comment:
np: Can we have all the case statements in the A-Z order?

Reply:
Yeah, can do